Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py

```diff
@@ -1004,10 +1007,11 @@ def _run_parallel_tests(self,
                 def make_parallel_test_func(test_spec, index, current_executable):
                     def run(request, workspace, extra_cmdline_args):
                         results = self._exec_single_test(
-                            request, workspace, current_executable, index + 1, self.log_name, test_spec, extra_cmdline_args)
-                        if results is None:
-                            raise EditorToolsFrameworkException(f"Results were None. Current log name is "
-                                                                f"{self.log_name} and test is {str(test_spec)}")
+                            request, workspace, current_executable, index + 1, self.log_name, test_spec,
+                            extra_cmdline_args)
+                        if not results:
```

Comment:
Is this a style we should be moving to? I almost always do `if var is None:`

Reply:
if X is None is fine, I just changed this to have the extra benefit of checking if the results dict is empty